luisy/config.py

```diff
@@ -48,12 +51,42 @@ def __init__(self):
         for param, env_key in env_keys.items():
             self.set_param(param, self._get_env_var(env_key))
 
-        if self.get_param('azure_storage_key') is not None:
+        self.init()
+
+    def _check_for_spark(self):
+        if SparkContext._active_spark_context:
+            return SparkSession.builder.getOrCreate()
+        else:
+            return self.get_param('spark')
```

Comment:
what happens if this fails ? We just get an exception that the key doe not exist ?